store/storetest/post_store.go

```diff
@@ -2148,6 +2148,11 @@ func testPostCountsByDay(t *testing.T, ss store.Store) {
 	r2, err = ss.Post().AnalyticsPostCount(&model.PostCountOptions{TeamId: t1.Id, ExcludeDeleted: true})
 	require.NoError(t, err)
 	assert.Equal(t, int64(5), r2)
+
+	// total users only posts for single team with the deleted post excluded
+	r2, err = ss.Post().AnalyticsPostCount(&model.PostCountOptions{TeamId: t1.Id, ExcludeDeleted: true, UsersPostsOnly: true})
+	require.NoError(t, err)
+	assert.Equal(t, int64(3), r2)
```

Comment:
In this test we only ignore system messages. 

Can we please make a bot post and confirm that the bot filtering works as well? This is a follow-up to my other comment about the typo in `UserId`.

Reply:
The test already creates two bots messages that gets filtered out.
If you look at the just above test case, the result there is 5 which includes those two messages from bot.